<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="167" started-at="2021-08-03T06:14:45Z" finished-at="2021-08-03T06:14:45Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="167" started-at="2021-08-03T06:14:45Z" finished-at="2021-08-03T06:14:45Z">
      <class name="testCases.IssueTestCases">
        <test-method status="FAIL" signature="Before()[pri:0, instance:testCases.IssueTestCases@4b1c1ea0]" name="Before" is-config="true" duration-ms="0" started-at="2021-08-03T11:44:45Z" finished-at="2021-08-03T11:44:45Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at utility.Constant.captureScreen_Negative(Constant.java:161)
	at pages.LoginPage.login(LoginPage.java:69)
	at testCases.IssueTestCases.Before(IssueTestCases.java:64)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:616)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Before -->
        <test-method status="SKIP" signature="BeforeClass()[pri:0, instance:testCases.IssueTestCases@4b1c1ea0]" name="BeforeClass" is-config="true" duration-ms="0" started-at="2021-08-03T11:44:45Z" finished-at="2021-08-03T11:44:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeClass -->
        <test-method status="SKIP" signature="Test040_UpdateIssue_AddNDeleteMat()[pri:0, instance:testCases.IssueTestCases@4b1c1ea0]" name="Test040_UpdateIssue_AddNDeleteMat" duration-ms="1" started-at="2021-08-03T11:44:45Z" finished-at="2021-08-03T11:44:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test040_UpdateIssue_AddNDeleteMat -->
        <test-method status="SKIP" signature="After()[pri:0, instance:testCases.IssueTestCases@4b1c1ea0]" name="After" is-config="true" duration-ms="0" started-at="2021-08-03T11:44:45Z" finished-at="2021-08-03T11:44:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- After -->
        <test-method status="SKIP" signature="AfterClass()[pri:0, instance:testCases.IssueTestCases@4b1c1ea0]" name="AfterClass" is-config="true" duration-ms="0" started-at="2021-08-03T11:44:45Z" finished-at="2021-08-03T11:44:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterClass -->
      </class> <!-- testCases.IssueTestCases -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
